frontend/src/pages/Status.tsx

```diff
@@ -55,6 +55,32 @@ export function hasFinished(status?: NodePhase): boolean {
   }
 }
 
+export function statusToFadedColor(status?: NodePhase): string {
+  if (!status) {
+    return color.fadedStatusColors.notStarted;
+  }
+
+  switch (status) {
+    case NodePhase.ERROR:
+      return color.fadedStatusColors.error;
+    case NodePhase.FAILED:
+      return color.fadedStatusColors.error;
+    case NodePhase.PENDING:
```

Comment:
nit: you can group and fall through all `notStarted` cases below.

Reply:
I'd prefer to keep `PENDING` separate since it's a legitimate case, unlike `UNKNOWN` and `undefined`